internal/app/go-filecoin/plumbing/msg/previewer.go

```diff
@@ -46,28 +42,32 @@ func NewPreviewer(chainReader previewerChainReader, cst *hamt.CborIpldStore, bs
 
 // Preview sends a read-only message to an actor.
 func (p *Previewer) Preview(ctx context.Context, optFrom, to address.Address, method types.MethodID, params ...interface{}) (types.GasUnits, error) {
-	encodedParams, err := abi.ToEncodedValues(params...)
-	if err != nil {
-		return types.NewGasUnits(0), errors.Wrap(err, "failed to encode message params")
-	}
+	// Dragons: delete
```

Comment:
Again is this suggesting doing away with message preview from plumbing or just this implementation of it?